### PR TITLE
👥 Irish translations

### PR DESCRIPTION
**Pull request by callumok2004**

### CODE REVIEW

File: websites/B/BanG Dream! Wikia/dist/metadata.json [hunk 1]
```diff
@@ -5,10 +5,11 @@
   },
   "service": "BanG Dream! Wikia",
   "description": {
-    "en": "Presence for the English Wikia of the BanG Dream! series"
+    "en": "Presence for the English Wikia of the BanG Dream! series",
+    "ga_IE": "Láithreacht don Wikia Béarla an aisling Bang!sraith"
   },
```
Comment (Bas950) [inline websites/B/BanG Dream! Wikia/dist/metadata.json:10]: Shouldn't there be a space after the !
```suggestion
    "ga_IE": "Láithreacht don Wikia Béarla an aisling Bang! sraith"
  },
```

File: websites/B/Bargrooves/dist/metadata.json [hunk 2]
```diff
@@ -8,10 +8,11 @@
     "www.bargrooves.live"
   ],
   "description": {
-    "en": "Bargrooves curates bespoke soundtracks for discerning house music fans; compilations and playlists to suit your seasonal moods, featuring the best current dance music alongside well-loved classics. Use this extension to show your friends what your listening to."
+    "en": "Bargrooves curates bespoke soundtracks for discerning house music fans; compilations and playlists to suit your seasonal moods, featuring the best current dance music alongside well-loved classics. Use this extension to show your friends what your listening to.",
+    "ga_IE": "Déanann Bargrooves coimeádaithe fuaimrian ordaithe le haghaidh lucht leanúna ceoil tithe géarchúisí;Tiomsúcháin agus seinmliostaí a oireann do do chuid mothúchán séasúrach, featuring an ceol damhsa is fearr in éineacht le clasaicí dea-ghrá.Bain úsáid as an síneadh seo chun do chairde a thaispeáint cad é atá uait."
   },
```
Comment (Bas950) [inline websites/B/Bargrooves/dist/metadata.json:13]:
```suggestion
    "ga_IE": "Déanann Bargrooves coimeádaithe fuaimrian ordaithe le haghaidh lucht leanúna ceoil tithe géarchúisí; Tiomsúcháin agus seinmliostaí a oireann do do chuid mothúchán séasúrach, featuring an ceol damhsa is fearr in éineacht le clasaicí dea-ghrá.Bain úsáid as an síneadh seo chun do chairde a thaispeáint cad é atá uait."
  },
```

File: websites/B/Blob Project/dist/metadata.json [hunk 3]
```diff
@@ -6,10 +6,11 @@
   },
   "url": "blob-project.com",
   "description": {
-    "en": "Hello, welcome to Blob Project. We are from Jakarta, Indonesia. We create an amazing project and are trusted by thousands of people."
+    "en": "Hello, welcome to Blob Project. We are from Jakarta, Indonesia. We create an amazing project and are trusted by thousands of people.",
+    "ga_IE": "Dia duit, Fáilte go dtí an tionscadal.Táimid ó Iacárta, an Indinéis.Cruthaímid tionscadal iontach agus tá muinín acu as na mílte duine."
   },
```
Comment (Bas950) [inline websites/B/Blob Project/dist/metadata.json:11]:
```suggestion
    "ga_IE": "Dia duit, Fáilte go dtí an tionscadal. Táimid ó Iacárta, an Indinéis.Cruthaímid tionscadal iontach agus tá muinín acu as na mílte duine."
  },
```

File: websites/B/Blockdit/dist/metadata.json [hunk 4]
```diff
@@ -6,10 +6,11 @@
   "service": "Blockdit",
   "description": {
     "en": "Blockdit is a social network platform that connects all great ideas together. Blockdit offers you remarkable stories, ideas, and knowledges in 'blocks' which are easy to read ",
-    "th": "Blockdit เป็นแพลตฟอร์มเครือข่ายสังคมออนไลน์ที่เชื่อมโยงความคิดที่ยอดเยี่ยมทั้งหมดเข้าด้วยกัน Blockdit นำเสนอเรื่องราวความคิดและความรู้ที่น่าทึ่งใน 'บล็อก' ซึ่งอ่านง่าย"
+    "th": "Blockdit เป็นแพลตฟอร์มเครือข่ายสังคมออนไลน์ที่เชื่อมโยงความคิดที่ยอดเยี่ยมทั้งหมดเข้าด้วยกัน Blockdit นำเสนอเรื่องราวความคิดและความรู้ที่น่าทึ่งใน 'บล็อก' ซึ่งอ่านง่าย",
+    "ga_IE": "Is ardán líonra sóisialta é Blocdit a nascann gach smaoineamh iontach le chéile.Tugann Blocdit scéalta iontacha, smaointe, agus knowledges duit i 'bloic' atá éasca le léamh"
   },
```
Comment (Bas950) [inline websites/B/Blockdit/dist/metadata.json:11]:
```suggestion
    "ga_IE": "Is ardán líonra sóisialta é Blocdit a nascann gach smaoineamh iontach le chéile. Tugann Blocdit scéalta iontacha, smaointe, agus knowledges duit i 'bloic' atá éasca le léamh"
  },
```

File: websites/B/Brony Radio Germany/dist/metadata.json [hunk 5]
```diff
@@ -5,10 +5,11 @@
   },
   "service": "Brony Radio Germany",
   "description": {
-    "en": "Brony Radio Germany is the web radio for German-speaking Bronies. We stream 24/7 music for everyone and it's all about ponies."
+    "en": "Brony Radio Germany is the web radio for German-speaking Bronies. We stream 24/7 music for everyone and it's all about ponies.",
+    "ga_IE": "Is é Brony Radio An Ghearmáin an raidió gréasáin do bhrúnna labhartha na Gearmáine.Sruthlaímid 24/7 ceol do gach duine agus tá sé go léir faoi phonies."
   },
```
Comment (Bas950) [inline websites/B/Brony Radio Germany/dist/metadata.json:10]:
```suggestion
    "ga_IE": "Is é Brony Radio An Ghearmáin an raidió gréasáin do bhrúnna labhartha na Gearmáine. Sruthlaímid 24/7 ceol do gach duine agus tá sé go léir faoi phonies."
  },
```

File: websites/B/BuzzFeed News/dist/metadata.json [hunk 6]
```diff
@@ -21,7 +21,8 @@
   ],
   "description": {
     "en": "BuzzFeed, Inc. is an American Internet media, news and entertainment company with a focus on digital media; it is based in New York City. BuzzFeed was founded in 2006 by Jonah Peretti and John S. Johnson III, to focus on tracking viral content.",
-    "tr": "BuzzFeed, Inc. dijital medyaya odaklanan bir Amerikan İnternet medyası, haber ve eğlence şirketidir; New York'ta dayanmaktadır. BuzzFeed, 2006 yılında Jonah Peretti ve John S. Johnson III tarafından viral içeriklerin izlenmesine odaklanmak üzere kuruldu."
+    "tr": "BuzzFeed, Inc. dijital medyaya odaklanan bir Amerikan İnternet medyası, haber ve eğlence şirketidir; New York'ta dayanmaktadır. BuzzFeed, 2006 yılında Jonah Peretti ve John S. Johnson III tarafından viral içeriklerin izlenmesine odaklanmak üzere kuruldu.",
+    "ga_IE": "BuzzFeed, Inc Is Meiriceánach Idirlín Meiriceánach, Nuacht agus siamsaíochta cuideachta le fócas ar na meáin dhigiteacha;Tá sé bunaithe i gCathair Nua-Eabhrac.Bhunaigh Jonah Peretti agus John S. Johnson III BUZZFEED i 2006, chun díriú ar ábhar víreasach a rianú."
   },
```
Comment (Bas950) [inline websites/B/BuzzFeed News/dist/metadata.json:26]:
```suggestion
    "ga_IE": "BuzzFeed, Inc Is Meiriceánach Idirlín Meiriceánach, Nuacht agus siamsaíochta cuideachta le fócas ar na meáin dhigiteacha; Tá sé bunaithe i gCathair Nua-Eabhrac.Bhunaigh Jonah Peretti agus John S. Johnson III BUZZFEED i 2006, chun díriú ar ábhar víreasach a rianú."
  },
```

File: websites/B/BuzzFeed/dist/metadata.json [hunk 7]
```diff
@@ -7,13 +7,14 @@
...
     "tr": "BuzzFeed, Inc. dijital medyaya odaklanan bir Amerikan İnternet medyası, haber ve eğlence şirketidir; New York'ta dayanmaktadır. BuzzFeed, 2006 yılında Jonah Peretti ve John S. Johnson III tarafından viral içeriklerin izlenmesine odaklanmak üzere kuruldu.",
-    "nl": "BuzzFeed, Inc. is een Amerikaans internetmedia-, nieuws- en entertainmentbedrijf met een focus op digitale media; het is gevestigd in New York City. BuzzFeed is in 2006 opgericht door Jonah Peretti en John S. Johnson III, om zich te concentreren op het volgen van virale inhoud."
+    "nl": "BuzzFeed, Inc. is een Amerikaans internetmedia-, nieuws- en entertainmentbedrijf met een focus op digitale media; het is gevestigd in New York City. BuzzFeed is in 2006 opgericht door Jonah Peretti en John S. Johnson III, om zich te concentreren op het volgen van virale inhoud.",
+    "ga_IE": "BuzzFeed, Inc Is Meiriceánach Idirlín Meiriceánach, Nuacht agus siamsaíochta cuideachta le fócas ar na meáin dhigiteacha;Tá sé bunaithe i gCathair Nua-Eabhrac.Bhunaigh Jonah Peretti agus John S. Johnson III BUZZFEED i 2006, chun díriú ar ábhar víreasach a rianú."
   },
```
Comment (Bas950) [inline websites/B/BuzzFeed/dist/metadata.json:12]:
```suggestion
    "ga_IE": "BuzzFeed, Inc Is Meiriceánach Idirlín Meiriceánach, Nuacht agus siamsaíochta cuideachta le fócas ar na meáin dhigiteacha; Tá sé bunaithe i gCathair Nua-Eabhrac.Bhunaigh Jonah Peretti agus John S. Johnson III BUZZFEED i 2006, chun díriú ar ábhar víreasach a rianú."
  },
```

File: websites/B/bilibili/dist/metadata.json [hunk 8]
```diff
@@ -14,7 +14,8 @@
     "en": "bilibili is a well-known video barrage website in China. Here is the most timely animation new fan, the best ACG atmosphere, the most creative Up master. Everyone can find a lot of joy here.",
     "nl": "bilibili is een bekende videowerversite in China. Hier is de meest actuele animatie nieuwe fan, de beste ACG-sfeer, de meest creatieve Up-meester. Iedereen kan hier veel plezier beleven.",
     "zh_CN": "bilibili是国内知名的视频弹幕网站，这里有最及时的动漫新番，最棒的ACG氛围，最有创意的Up主。大家可以在这里找到许多欢乐。",
-    "zh_TW": "bilibili是大陸知名的具有彈幕功能的影片串流網站，這裡有最即時的動漫新番，最棒的ACG氛圍，最有創意的實況主。大家可以在這裡找到許多歡樂。"
+    "zh_TW": "bilibili是大陸知名的具有彈幕功能的影片串流網站，這裡有最即時的動漫新番，最棒的ACG氛圍，最有創意的實況主。大家可以在這裡找到許多歡樂。",
+    "ga_IE": "Is láithreán gréasáin barrage físeán aitheanta é Bilibili sa tSín.Seo an lucht leanúna nua is tráthúla nua, an t-atmaisféar ACG is fearr, an máistir is cruthaitheora.Is féidir le gach duine a lán áthais a fháil anseo."
   },
```
Comment (Bas950) [inline websites/B/bilibili/dist/metadata.json:19]:
```suggestion
    "ga_IE": "Is láithreán gréasáin barrage físeán aitheanta é Bilibili sa tSín. Seo an lucht leanúna nua is tráthúla nua, an t-atmaisféar ACG is fearr, an máistir is cruthaitheora.Is féidir le gach duine a lán áthais a fháil anseo."
  },
```

File: websites/C/CartoonCrazy.net/dist/metadata.json [hunk 9]
```diff
@@ -6,13 +6,14 @@
   "service": "CartoonCrazy.net",
   "description": {
     "en": "Watch cartoons online at www.cartooncrazy.net and Stream 25000 Anime English dubbed in High Quality! You can watch anime and cartoons in ease.",
-    "nl": "Bekijk tekenfilms online op www.cartooncrazy.net en Stream 25000 Anime Engels nagesynchroniseerd in hoge kwaliteit! U kunt gemakkelijk anime en tekenfilms bekijken."
+    "nl": "Bekijk tekenfilms online op www.cartooncrazy.net en Stream 25000 Anime Engels nagesynchroniseerd in hoge kwaliteit! U kunt gemakkelijk anime en tekenfilms bekijken.",
+    "ga_IE": "Féachaint ar chartúin ar líne ag www.cartooncrazy.net agus sruth 25000 Anime Béarla dubbed i caighdeán ard!Is féidir leat féachaint ar anime agus cartúin ar a suaimhneas."
   },
```
Comment (Bas950) [inline websites/C/CartoonCrazy.net/dist/metadata.json:11]:
```suggestion
    "ga_IE": "Féachaint ar chartúin ar líne ag www.cartooncrazy.net agus sruth 25000 Anime Béarla dubbed i caighdeán ard! Is féidir leat féachaint ar anime agus cartúin ar a suaimhneas."
  },
```

File: websites/C/Chess/dist/metadata.json [hunk 10]
```diff
@@ -7,10 +7,11 @@
   "description": {
     "en": "Chess.com is the main internet chess server. It is also an Internet forum and a social network dedicated to chess.",
     "fr": "Chess.com est le principal serveur internet d'échecs. C'est également un forum internet et un réseau social consacré aux échecs.",
-    "es": "Chess.com es el principal servidor web para el ajedrez. Es también un foro web y un red social dedicado al ajedrez."
+    "es": "Chess.com es el principal servidor web para el ajedrez. Es también un foro web y un red social dedicado al ajedrez.",
+    "ga_IE": "Is é Chess.com an príomhfhreastalaí fichille Idirlín.Is fóram idirlín é freisin agus líonra sóisialta atá tiomnaithe do fichille."
   },
```
Comment (Bas950) [inline websites/C/Chess/dist/metadata.json:12]:
```suggestion
    "ga_IE": "Is é Chess.com an príomhfhreastalaí fichille Idirlín. Is fóram idirlín é freisin agus líonra sóisialta atá tiomnaithe do fichille."
  },
```

File: websites/C/Click Jogos/dist/metadata.json [hunk 11]
```diff
@@ -6,10 +6,11 @@
   "service": "Click Jogos",
   "description": {
     "en": "Founded in 2004, Click Jogos is now the largest online gaming portal in Brazil. All titles are free and do not require registration. Our goal is to provide an incredible entertainment experience.",
-    "pt_BR": "Fundado em 2004, o Click Jogos é hoje o maior portal de jogos online do Brasil. Todos os títulos são gratuitos e não exigem cadastro. Nosso objetivo é proporcionar uma experiência incrível em entretenimento."
+    "pt_BR": "Fundado em 2004, o Click Jogos é hoje o maior portal de jogos online do Brasil. Todos os títulos são gratuitos e não exigem cadastro. Nosso objetivo é proporcionar uma experiência incrível em entretenimento.",
+    "ga_IE": "Bunaithe i 2004, cliceáil Jogos anois an tairseach cearrbhachais is mó ar líne is mó sa Bhrasaíl.Tá na teidil go léir saor in aisce agus ní theastaíonn clárú uathu.Is é an sprioc atá againn ná taithí siamsaíochta dochreidte a sholáthar."
   },
```
Comment (Bas950) [inline websites/C/Click Jogos/dist/metadata.json:11]:
```suggestion
    "ga_IE": "Bunaithe i 2004, cliceáil Jogos anois an tairseach cearrbhachais is mó ar líne is mó sa Bhrasaíl. Tá na teidil go léir saor in aisce agus ní theastaíonn clárú uathu.Is é an sprioc atá againn ná taithí siamsaíochta dochreidte a sholáthar."
  },
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
I am guessing there should be spaces at these places, and you have it like 50/50 some have it some don't..  these are just a few.

**Comment by ririxidev** [comment]
squash commits ty thank

**Comment by callumok2004** [comment]
PR will be reopened when fixed whatever the translator script broke LOL

**Comment by Bas950** [comment]
> PR will be reopened when fixed whatever the translator script broke LOL

It didnt break lmao, I dont have the issue with Dutch, its just you guys not adding a space...

**Comment by ririxidev** [comment]
Wtf, translator script works fine lol

**Pull request closed without merging**